src/uwtools/tests/config/test_tools.py

```diff
@@ -296,6 +298,55 @@ def test_realize_config_output_file_conversion(tmp_path):
         assert f.read()[-1] == "\n"
 
 
+def test_realize_config_remove_nml_to_nml(tmp_path):
+    base = NMLConfig({"constants": {"pi": 3.141, "e": 2.718}})
+    s = """
+    constants:
+      e: !remove
+    """
+    loader = yaml.Loader(dedent(s).strip())
+    loader.add_constructor("!remove", UWYAMLRemove)
+    path = tmp_path / "config.nml"
+    assert not path.is_file()
+    tools.realize_config(
+        input_config=base,
+        output_file=path,
+        supplemental_configs=[YAMLConfig(loader.get_data())],
+    )
+    assert f90nml.read(path) == {"constants": {"pi": 3.141}}
```

Comment:
Just making sure here...the end user would not need to add their own UW constructors to use the api version of the `realize` tool, correct? This is just setting up an isolated case to test the new feature directly?

Reply:
Yes. This test confuses that, though. I'm going to update it to try to make that clearer.